### PR TITLE
ip64: Streamlined configuration parameters

### PR DESCRIPTION
**Pull request by uknoblic**
I just replaced the two defines "IP64_CONF_DHCP" and "IP64_CONF_UIP_FALLBACK_INTERFACE"
from ip64.c and moved their definition into ip64.h. In addition, i added IP64_CONF_DHCP to the example configuration file for completeness.

### CODE REVIEW

**Comment by vsaw** [comment]
The formatting makes it a bit hard to read but I think it looks good :+1:

File: core/net/ip64/ip64-conf-example.h [hunk 1]
```diff
@@ -35,9 +35,14 @@
...
-#define IP64_CONF_ETH_DRIVER             ip64_tap_driver
+#define IP64_CONF_ETH_DRIVER                ip64_tap_driver
 
-#endif /* IP64_CONF_H */
+/* 
+ * In contrast to the mandatory parameters above, IP64_CONF_DHCP is an 
+ * optional configuration parameter. The default value is set in ip64.h 
+ */
+/* #define IP64_CONF_DHCP                      1 */
+#ENDIF /* IP64_CONF_H */
```
Comment (simonduq) [inline core/net/ip64/ip64-conf-example.h]: uppercase endif?

**Comment by simonduq** [comment]
Hi, would be great if you could split this in two commits (or better, PRs), one for code style, one for the actual modification, to make reviewing (significantly) easier

**Comment by uknoblic** [comment]
Hi, i changed this pull-request to just contain the actual "streamlining" modifications for the configurations parameters. Should now be easy to review these small changes.

**Comment by simonduq** [comment]
:+1:

**Comment by laurentderu** [comment]
:+1:

**Pull request merged**